components/gems/Gemfile

```diff
@@ -17,9 +17,6 @@
 
 source "https://rubygems.org"
 
-# We found a segfault in ffi 1.13
-gem "ffi", "< 1.13.0"
```

Comment:
Can you also remove this pin from `omnibus/Gemfile`?

Reply:
Will do.

Reply:
Fixed